forge/forge/agent/agent_test.py

```diff
@@ -13,7 +13,7 @@
 from forge.file_storage.base import FileStorageConfiguration
 from forge.file_storage.local import LocalFileStorage
 
-from .agent import ProtocolAgent
+from ..agent_protocol.agent import ProtocolAgent
```

Comment:
```suggestion
from forge.agent_protocol.agent import ProtocolAgent
```

Reply:
I moved the `agent_test.py` and updated imports